.github/ISSUE_TEMPLATE/70_browsertesting_update.yml

```diff
@@ -0,0 +1,44 @@
+name: "Request Browser-Testing Dependencies Update"
```

Comment:
I think I'm not familiar w/ this pattern, why do we need both a .yml and .md file?

Reply:
We probably want both issue template and a scheduled workflow. Unfortunately I did not find a way to reference creation of the issue from workflow by template (flag exists but does not work for non-interactive scenarios). So i decided to reference a body-file of the issue that will be created by automated workflow.

Let me know if you think we can simplify it - I guess I just did not figure it out honestly

Reply:
i've removed the issue template - it will only be accessible via the workflow (which if works should be sufficient).